Project-I-Social-Card/social-card/src/components/HeaderComponents/HeaderContainer.js

```diff
@@ -1,2 +1,19 @@
 import React from 'react';
 import './Header.css';
+
+// import header contents
+import Thumbimg from '../HeaderComponents/ImageThumbnail';
+import MyTitle from '../HeaderComponents/HeaderTitle';
+import HeaderParagraph from '../HeaderComponents/HeaderContent';
+
+const HeaderContainer = () => {
+    return (
+        <div className='HeaderBox'>
+            <Thumbimg />
```

Comment:
Excellent use of 'container' components. This is why React is powerful: managing small components to build a larger app.